### PR TITLE
projects: max22200 : Add initial project files for MAX22200

### PR DESCRIPTION
**Pull request by RaduSabau1**
## Pull Request Description

#2015 needs to be merged before this PR.
Add initial common_data, examples and platform files for MAX22200 project.
Add necessary makefiles for MAX22200 application example build. Add builds.json file for MAX22200 application example build.
Application example runs on both MAX32666FTHR and AD-APARD32690-SL.

## PR Type
- [ ] Bug fix (change that fixes an issue)
- [x] New feature (change that adds new functionality)
- [ ] Breaking change (has dependencies in other repos or will cause CI to fail)

## PR Checklist
- [x] I have followed the [Coding style guidelines](http://analogdevicesinc.github.io/no-OS/drivers_guide.html#coding-style)
- [ ] I have performed a self-review of the changes
- [x] I have commented my code, at least hard-to-understand parts
- [x] I have build all projects affected by the changes in this PR
- [x] I have tested in hardware affected projects, at the relevant boards
- [x] I have signed off all commits from this PR
- [ ] I have updated the documentation (wiki pages, ReadMe etc), if applies

### CODE REVIEW

File: projects/max22200/src/common/common_data.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+#endif
+
+extern struct no_os_uart_init_param max22200_uart_ip;
+extern struct no_os_spi_init_param max22200_spi_ip;
+extern struct no_os_gpio_init_param max22200_cmd_ip;
+extern struct max22200_init_param max22200_ip;
+
+#endif /* __COMMON_DATA_H__ */
+
+
```
Comment (rbolboac) [inline projects/max22200/src/common/common_data.h:56]: nit: remove extra lines

**Review by rbolboac — APPROVED** [review]
just one nit from my side, other than that it looks good.

**Comment by RaduSabau1** [comment]
V2

- Removed extra lines.

**Comment by RaduSabau1** [comment]
V3

- Rebased with master.

**Pull request merged**